src/components/Task.tsx

```diff
@@ -26,12 +26,12 @@ const Task = ({
     : "";
 
   return (
-    <li className="todo-listItem" key={itemId} data-item-key={itemId}>
+    <li className="todo-listItem">
       <svg
         className="todo-listImage"
         strokeLinecap="round"
         strokeLinejoin="round"
-        onClick={() => completeTask(itemId)}
+        onClick={() => completeTask(id)}
```

Comment:
This is actually erroring at this point because we're still telling it in the props to expect a `Task` but we're passing it a string (`id`). We'll fix this too.